<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4VG4ENC" name="Home" tests="7" failures="5" timestamp="13 Jan 2023 07:21:58 GMT" time="161.66" errors="0">
  <testcase name="launchHomePage_Test" time="1.392" classname="com.automation.ui.digitaldeployer.testcases.index.IndexPageTest"/>
  <testcase name="verifyValidLogin" time="13.785" classname="com.automation.ui.digitaldeployer.testcases.login.LoginTest"/>
  <testcase name="launchAddNewWorkspacename_create_Test" time="18.307" classname="com.automation.ui.digitaldeployer.testcases.home.HomePageTest">
    <failure type="java.lang.AssertionError" message="ERROR WHILE EXECUTION ">
      <![CDATA[java.lang.AssertionError: ERROR WHILE EXECUTION 
	at com.automation.ui.digitaldeployer.pageobjectsfactory.pageobject.home.pages.HomePage.addws(HomePage.java:135)
	at com.automation.ui.digitaldeployer.testcases.home.HomePageTest.launchAddNewWorkspacename_create_Test(HomePageTest.java:102)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- launchAddNewWorkspacename_create_Test -->
  <testcase name="launchAddNewWorkspacename_create_Test" time="18.234" classname="com.automation.ui.digitaldeployer.testcases.home.HomePageTest">
    <failure type="java.lang.AssertionError" message="ERROR WHILE EXECUTION ">
      <![CDATA[java.lang.AssertionError: ERROR WHILE EXECUTION 
	at com.automation.ui.digitaldeployer.pageobjectsfactory.pageobject.home.pages.HomePage.addws(HomePage.java:135)
	at com.automation.ui.digitaldeployer.testcases.home.HomePageTest.launchAddNewWorkspacename_create_Test(HomePageTest.java:102)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- launchAddNewWorkspacename_create_Test -->
  <testcase name="launchAddNewWorkspacename_create_Test" time="18.329" classname="com.automation.ui.digitaldeployer.testcases.home.HomePageTest">
    <failure type="java.lang.AssertionError" message="ERROR WHILE EXECUTION ">
      <![CDATA[java.lang.AssertionError: ERROR WHILE EXECUTION 
	at com.automation.ui.digitaldeployer.pageobjectsfactory.pageobject.home.pages.HomePage.addws(HomePage.java:135)
	at com.automation.ui.digitaldeployer.testcases.home.HomePageTest.launchAddNewWorkspacename_create_Test(HomePageTest.java:102)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- launchAddNewWorkspacename_create_Test -->
  <testcase name="launchAddNewWorkspacename_create_Test" time="18.263" classname="com.automation.ui.digitaldeployer.testcases.home.HomePageTest">
    <failure type="java.lang.AssertionError" message="ERROR WHILE EXECUTION ">
      <![CDATA[java.lang.AssertionError: ERROR WHILE EXECUTION 
	at com.automation.ui.digitaldeployer.pageobjectsfactory.pageobject.home.pages.HomePage.addws(HomePage.java:135)
	at com.automation.ui.digitaldeployer.testcases.home.HomePageTest.launchAddNewWorkspacename_create_Test(HomePageTest.java:102)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- launchAddNewWorkspacename_create_Test -->
  <testcase name="launchAddNewWorkspacename_create_Test" time="18.351" classname="com.automation.ui.digitaldeployer.testcases.home.HomePageTest">
    <failure type="java.lang.AssertionError" message="ERROR WHILE EXECUTION ">
      <![CDATA[java.lang.AssertionError: ERROR WHILE EXECUTION 
	at com.automation.ui.digitaldeployer.pageobjectsfactory.pageobject.home.pages.HomePage.addws(HomePage.java:135)
	at com.automation.ui.digitaldeployer.testcases.home.HomePageTest.launchAddNewWorkspacename_create_Test(HomePageTest.java:102)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- launchAddNewWorkspacename_create_Test -->
</testsuite> <!-- Home -->
